src/ducks/files/Toolbar.jsx

```diff
@@ -56,8 +56,8 @@ const mapStateToProps = (state, ownProps) => ({
 })
 
 const mapDispatchToProps = (dispatch, ownProps) => ({
-  uploadFile: (file, displayedFolder) => {
-    dispatch(uploadFile(file, displayedFolder))
+  uploadFiles: (files, displayedFolder) => {
+    dispatch(uploadFiles(files, displayedFolder))
```

Comment:
It will be clearer to rename that 2 variables as we can miss than the second comes from an import.